test/OpenTelemetry.Tests/Trace/TracerProviderSdkTest.cs

```diff
@@ -369,9 +369,10 @@ public void TracerProvideSdkCreatesAndDiposesInstrumentation()
         }
 
         [Fact]
-        public void TracerProviderSdkBuildsWithDefaultResource()
+        public void TracerProviderSdkBuildsWithSDKResource()
```

Comment:
This test is expected to validate that when TracerProvider is built without explicitly adding any resource, it should still create a default resource. We need to retain this test same for most parts, except validate that a single resource attribute is available in the tracerProvider.GetResource(), and it should contain the service.name "unknown service..."

Reply:
Sounds good, kept this as an extra test as it checks that we can add SDK resource properly. Re-added original test that just builds with no extra chaining and checks that it contains a single attribute.